### PR TITLE
Neural networks

### PR DESCRIPTION
**Pull request by nicholassheffler**
Ready for review

### CODE REVIEW

File: NeuralNetworks/Practice/Questions.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<br>
+
+A GPU can be used to increase the speed at which the neural network trains.
+
+</details>
+
+---
+
+<details>
+<summary>What line of code is used to ensure that your user has access to the GPU?</summary>
```
Comment (PinkNoize) [inline NeuralNetworks/Practice/Questions.md]: "line of code" seems kinda misleading, maybe "command" instead
Also I don't know if this question is appropriate for this section as it is about setting up ROCm and not neural networks

File: NeuralNetworks/Practice/Questions.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+</details>
+
+---
+
+<details>
+<summary>What libraries are recommended for doing neural network applications?</summary>
+<br>
+
+Tensorflow and Keras.
```
Comment (PinkNoize) [inline NeuralNetworks/Practice/Questions.md]: PyTorch?

File: NeuralNetworks/Practice/Questions.md (hunk 1 above)
Comment (nicholassheffler) [reply NeuralNetworks/Practice/Questions.md]: I changed it to command. I think it is important as its the only part that actually relates to the GPU kit.

**Comment by PinkNoize** [comment]
Looks good to me

**Pull request merged**